test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -2045,7 +2045,7 @@ BOOST_AUTO_TEST_CASE(contracts_as_addresses)
 		}
 		contract test {
 			helper h;
-			function test() { h = new helper(); h.send(5); }
+			function test() payable { h = new helper(); h.send(5); }
```

Comment:
Yes. I've only added the keyword to the tests where it was failing without the keyword and I've reviewed that the code actually needs it.